Src/Support/Google.Apis.Auth/OAuth2/GoogleWebAuthorizationBroker.cs

```diff
@@ -34,22 +35,10 @@ namespace Google.Apis.Auth.OAuth2
     /// </remarks>
     public class GoogleWebAuthorizationBroker
     {
-        // It's unforunate this is a public field. But it cannot be changed due to backward compatibility.
-        /// <summary>The folder which is used by the <see cref="Google.Apis.Util.Store.FileDataStore"/>.</summary>
-        /// <remarks>
-        /// The reason that this is not 'private const' is that a user can change it and store the credentials in a
-        /// different location.
-        /// </remarks>
-        public static string Folder = "Google.Apis.Auth";
-
         /// <summary>
         /// Asynchronously authorizes the specified user.
         /// Requires user interaction; see <see cref="GoogleWebAuthorizationBroker"/> remarks for more details.
         /// </summary>
-        /// <remarks>
-        /// In case no data store is specified, <see cref="Google.Apis.Util.Store.FileDataStore"/> will be used by 
```

Comment:
Hmmm, I'm not convinced it's a good idea to not provide a data store by default. I think FileDataStore might be a good candidate to move to that very light common library.

Reply:
Possibly. I'd really hoped to get away without any concrete classes - and more importantly, FileDataStore uses  NewtonsoftJsonSerializer.Instance :(